required/latex-lab/latex-lab-math.dtx

```diff
@@ -287,197 +288,334 @@
+%   \item[consumer support] Currently only ngpdf makes
+%  use of it: if there is no mathml it passes the source to mathjax.
+%   \end{description}
+%
+%  \item[luamml] The following options make (with lualatex) use
+%  of the \pkg{luamml} package. \pkg{luamml} is currently automatically
+%  loaded (at the end of the preamble) if \pkg{unicode-math} has been detected.
+%  The loading can be forced or suppressed
+%  with \verb+\tagpdfsetup{math/mathml/luamml/load=true/false}+.
+%
+%  \pkg{luamml} affects all math, locally it can be stopped with |math/mathml/ignore|,
+%  or by using the commands described in the package.
+%
+%  \item[mathml-AF] A mathml representation of the equation can be attached
+%  to the structure. The options are rather complex as the
+%  keys have to control three different  tasks:
+%  The \emph{generation} of the file with the mathml fragments,
+%  the \emph{reading} and \emph{embedding} of the mathml fragments,
+%  and the \emph{association} of a mathml fragment to a specific equation.
+%
+%  \begin{description}
+%  \item[generation]
+%   With pdf\LaTeX{} mathml fragments can not be generated automatically,
+%   but a file with dummy fragments for every equation will be written if
+%   \verb+\tagpdfsetup{math/mathml/write-dummy}+ is issued.
+%
+%   With lua\LaTeX{} a file with mathml fragments will be created automatically
+%   if the package \pkg{luamml} has been loaded (see above).
+%
+%   \item[reading and embedding]
+%    By default the code will read and embed
+%    mathml from |\jobname-mathml.html| and |\jobname-luamml-mathml.html|.
+%    The list of sources can be changed with the key
+%    |math/mathml/sources|, setting that to an empty value suppresses
+%    the loading mathml associated files completely. For efficiency reasons
+%    if embeds math fragments directly, there is no check yet if the fragment is
+%    actually used.
+%
+%    The files are by default shown in the EmbeddedFiles pane,
+%    this can be disabled with |viewer/pane/mathml=false|.
+%
+%   \item[attaching] A mathml fragment is currently
+%   attached as associated file to an Formula if the hash of
+%   the source matches the hash of the fragment. This is not a perfect test:
+%   equations with the same source and so the same hash
+%   can have different mathml representation, e.g.
+%   if there are references or commands or counters in the equation.
+%   The attachment can be suppressed locally with |math/mathml/AF=false|.
+%   The mathml fragment will still be embedded in the PDF!
+%
+%  \end{description}
+%
+%  \item[mathml structure elements]
+%  Mathml structure elements can be used in PDF 2.0 directly.
+%  In PDF 1.7. one could theoretically
+%  use them if one declares a role mapping first,
+%  (this can be done with \verb+\tagpdfsetup{role/mathml-tags}+)
+%  which maps all to \texttt{Span}. But such a role mapping currently breaks reading
+%  e.g. in Adobe and so it is not recommended.
```

Comment:
generally (in many files not just here)  "foo e.g. bar"  is better written as  "foo, e.g., bar ....,"